scripts/runAccTests.sh

```diff
@@ -2,32 +2,63 @@
 set -e
 
 log() {
-  echo ""
-  echo "##################################"
-  echo "-------> $1"
-  echo "##################################"
+  echo "####################"
+  echo "## ->  $1 "
+  echo "####################"
 }
 
 setup() {
-  export DOCKER_REGISTRY_ADDRESS="127.0.0.1:15000"
-  export DOCKER_REGISTRY_USER="testuser"
-  export DOCKER_REGISTRY_PASS="testpwd"
-  export DOCKER_PRIVATE_IMAGE="127.0.0.1:15000/tftest-service:v1"
-  sh "$(pwd)"/scripts/testing/setup_private_registry.sh
+  # Create self signed certs
+  mkdir -p "$(pwd)"/scripts/testing/certs
+  openssl req \
+    -newkey rsa:2048 \
+    -nodes \
+    -x509 \
+    -days 365 \
+    -subj "/C=US/ST=Denial/L=Springfield/O=Dis/CN=127.0.0.1" \
+    -keyout "$(pwd)"/scripts/testing/certs/registry_auth.key \
+    -out "$(pwd)"/scripts/testing/certs/registry_auth.crt
+  # Create auth
+  mkdir -p "$(pwd)"/scripts/testing/auth
+  # Start registry
+  docker run --rm --entrypoint htpasswd registry:2 -Bbn testuser testpwd > "$(pwd)"/scripts/testing/auth/htpasswd
+  docker run -d -p 15000:5000 --rm --name private_registry \
+    -v "$(pwd)"/scripts/testing/auth:/auth \
+    -e "REGISTRY_AUTH=htpasswd" \
+    -e "REGISTRY_AUTH_HTPASSWD_REALM=Registry Realm" \
+    -e "REGISTRY_AUTH_HTPASSWD_PATH=/auth/htpasswd" \
+    -v "$(pwd)"/scripts/testing/certs:/certs \
+    -e "REGISTRY_HTTP_TLS_CERTIFICATE=/certs/registry_auth.crt" \
+    -e "REGISTRY_HTTP_TLS_KEY=/certs/registry_auth.key" \
+    registry:2
+  # wait a bit for travis...
+  sleep 5
+  # Login to private registry
+  docker login -u testuser -p testpwd 127.0.0.1:15000
+  # Build private images
+  for i in $(seq 1 3); do 
+    docker build -t tftest-service --build-arg JS_FILE_PATH=server_v${i}.js "$(pwd)"/scripts/testing -f "$(pwd)"/scripts/testing/Dockerfile
```

Comment:
By curiosity, is there a reason for this specific Dockerfile with `nodejs` ?

Reply:
Initially, I started setting up the `service` resource for a `nodejs` project and I wanted to test specific mounting and updating of configs/secrets. That's why I chose it.